### ISSUE TITLE
Document how to properly build and deploy a patch so Skotty is not the only person able to do this

### ISSUE THREAD

**Issue by Caenen** [Freelancer-Sirius-Revival/FLSR#916]
If time is not scarce that day/evening, this could be done in parallel with preparing the next major patch.

**Comment by erikroe** [comment]
Duplicate of https://github.com/Freelancer-Sirius-Revival/FLSR/issues/175
